dash/_callback.py

```diff
@@ -331,9 +337,12 @@ def wrap_func(func):
         def add_context(*args, **kwargs):
             output_spec = kwargs.pop("outputs_list")
             app_callback_manager = kwargs.pop("long_callback_manager", None)
-            callback_ctx = kwargs.pop("callback_context", {})
+            callback_ctx = kwargs.pop(
+                "callback_context", AttributeDict({"updated_props": {}})
```

Comment:
Why does `callback_ctx` always need to contain an `updated_props` key?

Reply:
I remember there is an error with some mocked tests.